tests/ml_commons/test_ml_commons_client.py

```diff
@@ -183,6 +183,13 @@ def test_integration_model_train_upload_full_cycle():
                     print("Model Task Status:", ml_task_status)
                     raised = True
                 assert raised == False, "Raised Exception in pulling task info"
+
+                try:
+                    search_task_obj = ml_client.search_task(_id=task_id)
```

Comment:
can we try to search in other ways also?

In testing, we usually try to see how other ways we can search to make sure this search_task is functioning accordingly.